Unix/doc/setup-kerberos-omi.md

```diff
@@ -651,8 +655,25 @@ TBD
 
  - Cause 1: Wrong domain/user/password on command line
```

Comment:
I know you didn't set it up this way, but please reformat to give this headings, then add them to TOC at the top of contents. Common errors should be easily found in the table of contents.